specification/azurestackhci/resource-manager/Microsoft.AzureStackHCI/stable/2022-10-01/updateSummaries.json

```diff
@@ -0,0 +1,213 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-10-01",
+    "title": "AzureStackHCI",
+    "description": "Azure Stack HCI management service"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AzureStackHCI/clusters/{clusterName}/updateSummaries/default": {
```

Comment:
>default [](http://example.com/codeflow?start=145&length=7)

Current guidance is to model this as an enum parameter with one value, rather than hard-coded path segment.

Reply:
For current resources in our RP, we use default in the path (as a non-enum) and would prefer to keep it that way if possible